package.json

```diff
@@ -21,9 +21,19 @@
   "main": "./dist/js/okta-sign-in.entry.js",
   "scripts": {
     "clean": "rimraf target && rimraf dist && rimraf build2",
-    "lint:report": "grunt lint && stylelint assets/sass/ > build2/OSW-stylelint-checkstyle.xml --custom-formatter node_modules/stylelint-checkstyle-formatter || true && eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml . || true",
-    "lint": "grunt lint && stylelint assets/sass/ && eslint .",
-    "test:dev": "grunt generate-config && karma start",
+    "generate-phone-codes": "node buildtools generate-phone-codes",
+    "generate-config": "node buildtools generate-language-config",
+    "generate-jsonp": "node buildtools generate-language-jsonp",
+    "retirejs": "retire --jspath src -p",
+    "lint:eslint": "eslint .",
+    "lint:eslint:report": "eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml .",
+    "lint:stylelint": "stylelint assets/sass/",
+    "lint:stylelint:report": "stylelint --custom-formatter node_modules/stylelint-checkstyle-formatter assets/sass/ > build2/OSW-stylelint-checkstyle.xml",
+    "lint:grunt": "grunt lint",
+    "lint:grunt:report": "grunt lint --checkstyle",
+    "lint": "mkdir build2 && run-p -c lint:eslint lint:grunt lint:stylelint",
+    "lint:report": "mkdir build2 && run-p -c lint:eslint:report lint:grunt:report lint:stylelint:report",
```

Comment:
we are generating individual reports right? Will this affect bacon? I know we run our tests in travis, but might be worth checking if bacon was also looking for some report...

Reply:
looks like it's not working for a while and added the fix.